Dockerfile.installer-image-build

```diff
@@ -1,14 +1,13 @@
 #This image contains the fedora livecd.sio
 FROM quay.io/ocpmetal/livecd-iso:fedora-coreos-livecd AS livecd-build
 
-FROM python:3
+FROM python:3.7-alpine
 ARG WORK_DIR=/data
 
 RUN mkdir $WORK_DIR
 RUN chmod 777 $WORK_DIR
 
 COPY --from=livecd-build /root/image/livecd.iso $WORK_DIR
-RUN chmod 666 $WORK_DIR/livecd.iso
```

Comment:
Don't we need to change the permissions of the livecd.iso?
If we do, we can just do it in the first layer

Reply:
@eranco74 i am not sure you can remove chmod for livecd.iso. chmod was added specifically so that we would be able to run on OCP. I like Ronny permission to run chmod in the first image

Reply:
Updated